source/extensions/filters/listener/tls_inspector/tls_inspector.cc

```diff
@@ -34,9 +34,14 @@ Config::Config(
     uint32_t max_client_hello_size)
     : stats_{ALL_TLS_INSPECTOR_STATS(POOL_COUNTER_PREFIX(scope, "tls_inspector."))},
       ssl_ctx_(SSL_CTX_new(TLS_with_buffers_method())),
-      enable_ja3_fingerprinting_(proto_config.enable_ja3_fingerprinting()),
       max_client_hello_size_(max_client_hello_size) {
 
+  if (proto_config.has_enable_ja3_fingerprinting()) {
```

Comment:
There's a helper macro for this pattern. Put back in the initializer list as `enable_ja3_fingerprinting_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(proto_config, proto_config.enable_ja3_fingerprinting, false)`

Reply:
Updated to use macro in commit 4315458fd65b361c208b5fddd2ac6a7854989354